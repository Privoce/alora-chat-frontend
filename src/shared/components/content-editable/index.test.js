import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import {
	ContentEditableComponent
} from 'shared/components';

describe('<ContentEditableComponent />', () => {
	const modelI = {
		isFetching: false
	};

	const getWrapper = (model) => {
		return shallow((
			<div>
				<ContentEditableComponent {...model} />
			</div>
		));
	};

	it('renders correctly', () => {
		const wrapper = getWrapper(modelI);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
