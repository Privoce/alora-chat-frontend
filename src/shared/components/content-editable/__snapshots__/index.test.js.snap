// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContentEditableComponent /> renders correctly 1`] = `
<div>
  <ContentEditableComponent
    isFetching={false}
  />
</div>
`;
